### ISSUE TITLE
very bugged with latest windows version

### ISSUE THREAD

**Issue by F4bi05** [ChrisAnd1998/TaskbarXI#185]
With the Windows version released the 26 september it cut half of the taskbar

**Comment by pdaddyo** [comment]
Yep same here, half my icons and start button cut off, seems to be calculating width wrong (since they added the copilot button perhaps?)

![image](https://github.com/ChrisAnd1998/TaskbarXI/assets/7074964/2c4cee7b-a53b-4b8a-94a9-9af9618f3135)

![image](https://github.com/ChrisAnd1998/TaskbarXI/assets/7074964/80854097-8b69-4e9f-b560-684d975dd416)

**Comment by F4bi05** [comment]
yeah probably microsoft touched something that broke the calculation of the width. I really hope in an update of taskbarXI that fix this, or windows, i just want my cool taskbar back.

**Comment by beuh0623** [comment]
Yeah I just got this. Money hungry devs aren't gonna do anything to fix it. Just delete

**Comment by bblicke1** [comment]
> Yeah I just got this. Money hungry devs aren't gonna do anything to fix it. Just delete

Yea, damn money hungry devs.... making apps that cost nothing, what is wrong with them?!
